template/deploy.sh

```diff
@@ -0,0 +1,31 @@
+# You will need the heroku-cli:
+# https://devcenter.heroku.com/articles/heroku-cli
+
```

Comment:
I don't think we can set -e. Some of these commands will fail if the setup script is run again and heroku will show errors appropriate errors along the way.

Reply:
That makes sense. Usually I prefer to be explicit about something like that. You can write `whatever command || true` to prevent the script from aborting if `whatever command` fails.